.c3i/authorized_users.yml

```
@@ -1359,3 +1359,7 @@ authorized_users:
 - tttapa
 - zeeshancs07
 - jll63
+- Jie-Cheng
+- jjardina
+- soonick
+- Mercotui
```

```suggestion
- Mercotui
- praetorian20
```

Asked yesterday and have an open PR.